topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md

```diff
@@ -0,0 +1,668 @@
+
+A Galaxy workflow can be registered by anyone on the WorkflowHub and Dockstore by following their documentation. In this section, we will see how to register a workflow on both WorkflowHub and Dockstore (Tip 1 - *"Register the workflow."*) by making it available in a public GitHub repository (Tip 3 - *"Make source code available in a public code repository."*) maintained by the Intergalactic Workflow Commission (IWC), a Galaxy community effort.
+
+The IWC maintains high-quality Galaxy Workflows via a [Galaxy Workflows Library](https://iwc.galaxyproject.org/). All workflows are reviewed and tested before publication and with every new Galaxy release. Deposited workflows follow best practices and are versioned using GitHub releases. Workflows also contain important metadata. Additionally the IWC collects further best practices, tips and tricks, FAQs and assists the community in designing high-quality Galaxy workflows.
+
+IWC offer [guidelines for adding workflow](https://github.com/galaxyproject/iwc/blob/main/workflows/README.md#adding-workflows) that will go through:
+
+1. Check for workflow eligibility
+2. Ensure workflows follow best-practices (already done)
+3. Generate tests
+
+## Check for workflow eligibility
+
+IWC collects production workflows targeted at users that want to analyze their own data. As such, the workflow should be sufficiently generic that users can provide their own data.
+
+IWC encourage, but do not require, links to related Galaxy Training Network Tutorials. Importantly, each workflow should be described in a way that a user can run the workflow on their own data without modifying the workflow. If we want to deposit a workflow that accompanies a tutorial we have to make sure that the workflow does not refer to datasets that only make sense in the context of the tutorial.
+
+By fulfilling the first Tips as we did before, we guarantee for workflow eligibility.
+
+## Generate tests
+
+This is usually the most difficult part and we encourage all new contributors to IWC to propose their workflows even if they did not managed to generate tests. However, the publication of these workflow will be speed up if tests are already present. 
+
+### Find input datasets
+
+To test the workflow, we need input datasets. By fulfilling Tip 4 (*"Provide example input data and results along with the workflow."*), we generated a toy dataset. We now need to publish it to [**Zenodo**](https://zenodo.org/) to have a permanent URL, also allowing others to **easily retrieve and reuse the data** when running or validating the workflow.
```

Comment:
In https://github.com/galaxyproject/training-material/pull/6119/files#diff-73ef156f75f9e4388f6473b70c13db1b5d26115d1f1007db168182ad6e858725R335 we already had test data?

Should we move this paragraph up?